VRC.Car.Pi/car.py

```diff
@@ -0,0 +1,17 @@
+from time import sleep
+from hardware_controller import HardwareController
+from messaging_handler import MessagingHandler
+
+hardwareController = HardwareController()
+messagingHandler = MessagingHandler(hardwareController)
+messagingHandler.connect()
+
+# Main loop
+while True:
+  print("Color Sensor: " + str(hardwareController.read_color_sensor()))
+  print("Ultrasonic Sensor: " + str(hardwareController.read_ultrasonic_sensor()))
+  print("Light Sensor: " + str(hardwareController.read_light_sensor()))
+  sleep(5)
+  # hardwareController.set_buzzer(1)
```

Comment:
moet deze commentaar blijven staan of is die vergeten weg te halen?

Reply:
yes das test code

Reply:
okide